force_wfmanager/ui/review/plot.py

```diff
@@ -122,11 +119,12 @@ class BasePlot(BaseDataView):
     )
 
     def _plot_updater_default(self):
-        return Timer(1000, self._update_plot_data)
+        return Timer(1000, self._check_scheduled_updates)
 
     def __plot_default(self):
         self._plot = self.plot_scatter()
-        self.resize_plot()
+        self.recenter_plot()
+        self.plot_updater.Start()
         return self._plot
 
     def _get_scatter_inspector_overlay(self, scatter_plot):
```

Comment:
UX issue: This might be being called when the color plot radio button is being selected, since it removes the axes titles.

The titles will only appear again if either `x` or `y` variables are changed

Reply:
This code snippet should refer to `_get_scatter_inspector_overlay` rather than `__plot_default`